<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="css/gb.css">
    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;700&display=swap">
    <link rel="icon" type="image/x-icon" href="image/favicon.png">
    <title>Gertraud Burger | research and collaborations</title>
</head>
<body class="top_page">
    <header>

        <div class="topbar_2">
            <div class="gblab_2">
                <a class="home_link" href="home.html">
                  <span class="big">G</span>ERTRAUD <span class="big">B</span>URGER |
                  <br>research and collaborations
                </a>
            </div>
            <button class="menu-toggle" id="menuToggleBtn" onclick="toggleMenu()">☰</button>
            <nav class="navbar2">
                <a href="home.html">Home</a>
                <a href="current_research.html">Current Research</a>
                <a href="publications_and_lectures.html">Publications and Lectures</a>
                <a href="people.html">Lab members</a>
                <a href="home.html#personal_information">Personal Information</a>
                <a href="teaching.html">Teaching</a>
            </nav>
        </div>

        <div class="bottom_nav">
            <div class="overlay"></div>
            <div class="part"><h1>Current Research</h1></div>
        </div>

    </header>

    <div class="page_research">

        <div class="current_research">
            
            <h2 class="secondary_pages"> Current projects</h2>
            <div class="current_projects">
                <div class="theme">
                    <img src="image/th1.png" alt="image1">
                    <p class="project_name">
                        Interactions between plants and endosymbiotic fungi and bacteria: 'Omics study of beneficial microbial symbionts of cranberry (MycAtok project).
                    </p>
                </div>
                <p class="project_text">
                    Numerous plants host inside their cells fungi and bacteria. Some of these microbes suppress plant pathogens (biocontrol), deliver nitrogen and phosphorus to their host, modulate plant growth, and act on the metabolism and stress response of the plant.
                    <br>
                    The goal of this project is to isolate endosymbiotic microbes from plants and screen the former for biocontrol and plant growth stimulation, with a particular emphasis on complementarity and synergy between fungal and bacterial endosymbionts. We will sequence the microbes' genomes, assemble them, and perform functional annotation to identify genes with the potential of anti-microbial action and mediating interspecfic molecular crosstalk. Transcriptomics will help to unravel the molecular basis of microbe-microbe and microbe-plant interactions. The system of choice is the Ericacean plant <i>Vaccinium macrocarpon</i>, the North American cranberry, and its natural endosymbionts. 
                    <br>
                    MycAtok is a collaborative project, in which participate academics, agronomists and farmers, and which involves also metabolomics and microbiology approaches, as well as greenhouse and field trials. (NSERC CRDPJ 514188-17)
                </p>

                <div class="theme">
                    <img src="image/th2.png" alt="image2">
                    <p class="project_name">
                        Unconventional genome architecture, gene structure and gene expression in mitochondria of micro-eukaryotes
                    </p>
                </div>
                <p class="project_text">
                    We are studying the extraordinary biological diversity outside the 'main-stream' eukaryotic phyla.
                    <br>
                    The focus of our current work is on a eukaryotic group displaying most unusual gene expression: the diplonemids.
                    In <i>Diplonema papillatum</i>, we discovered that its mitochondrial genome consists of ~80 different
                    circular chromosomes, each of which carries a small piece of a gene (for a review see e.g., Valach et al 2016 RNA Biol). 
                    Gene modules are transcribed separately and the resulting transcripts are then joined together to contiguous mRNAs and rRNAs by a process we refer to as trans-splicing. 
                    <br>
                    Further, we detected several two types of RNA editing, affecting about 2/3 of genes. One proceeds by insertion of multiple Us at a given site, the other by nucleotide substitutions (C to U and A to I). Other diplonemid species (e.g. <i>D. ambulator</i>, <i>D.</i> sp. 2, <i>Rhynchopus euleides</i>) have also a multipartite mitochondrial genome, gene fragmentation und RNA editing - though with resourceful variations on the theme.
                    <br>
                    Our working hypothesis is that trans-splicing and RNA editing in diplonemids
                    is directed by trans-factors, probably proteins. To test this hypothesis, we are employing genomics, transcriptomics, proteomics and 
                    bioinformatics approaches. Experimental data help to formulate specific search strategies
                    and vice versa, bioinformatics hypotheses are being tested experimentally.
                    <br>
                    In addition, we have started exploring mtDNAs of other members of Euglenozoa, notably 
                    yet undescribed diplonemids and euglenids, in order to
                    trace back the evolution of multipartite mitochondrial genomes, gene fragmentation, and RNA editing in this lineage (Kaur et al 2020 Nucleic Acids Res). (NSERC RGPIN-2019-04024)
                </p>

                <div class="theme">
                    <img src="image/th7.png" alt="image3">
                    <p class="project_name">
                        Unraveling novel molecular functions
                    </p>
                </div>
                <p class="project_text">
                    To get insight into the machineries that catalyze the unique trans-splicing and RNA editing in diplonemid mitochondria, and also to infer the metabolic capacity of these protists, we have sequenced the nuclear genome and transcriptome of <i>Diplonema papillatum</i>. Detailed gene-function annotion is underway. Collaborators in this project are Julius Lukes (University of South Bohemia, CZ), and since recently, Tom Williams (University of Bristol, UK), with genome sequencing, assembly and automated function-annotation spearheaded by the Burger laboratory (Valach et al 2023, BCM Biol). (FRQNT 2023-PR-326068)
                </p>

                <div class="theme">
                    <img src="image/th3.png" alt="image4">
                    <p class="project_name">
                        New genetic tools for marine micro-eukaryotes
                    </p>
                </div>
                <p class="project_text">
                    Unlike baker's yeast and other model organisms, there are no genetics or reverse-genetics tools available for diplonemids. In collaboration with J. Lukes (University of South Bohemia, CZ) and P. Keeling (University of British Columbia, CA), we are about to overcome this limitation. Methods are being developped for replacing, knocking-out, and tagging <i>Diplonema</i> genes, with the first hurdle successfully taken: the stable introduction and integration of foreign DNA in Diplonema's nuclear genome. Once in place, these new methods will substantially facilitate and accelerate the study of the unique molecular-mechanistic abilities of this organismal group (Faktorova et al 2020, Environ Microbiol). (Gordon and Betty Moore Foundation, GBMF MMI GBMF4983.01)
                </p>
                
                <div class="theme">
                    <img src="image/th4.png" alt="image5">
                    <p class="project_name">
                        Reconstruction of the ancestral nuclear genome
                    </p>
                </div>
                <p class="project_text">
                    The protist group called jakobids is considered to include the most primitive (least derived or most ancestral) eukaryotes known. To understand the basic gene make-up of primitive eukaryotes, we are sequencing nuclear genomes from this group. These data will also help to resolve with confidence which group the earliest offshoot of eukaryotes may be. So far, this question has not been resolved with confidence, due to the extremely deep divergences in the eukaryotic phylogenetic tree. <br>
                    <br>
                    As of now, we have draft assemblies from <i>Andalucia godoyi</i> and <i>Reclinomonas americana</i>, which are in the process of being completed and function-annotated. The 'Jakobids Genome Consortium' includes, in addition to Burger's group, BF. Lang (UdeM), M. Elias (Ostrava University, CZ), MW. Gray and A. Roger (both Dalhousie University, Halifax, CA) (Gray et al 2019 BMC Biol). (NSERC RGPIN-2014-05286)
                </p>

                <div class="theme">
                    <img src="image/th5.png" alt="image6">
                    <p class="project_name">
                        Information processing in primitive eukaryotes
                    </p>
                </div>
                <p class="project_text">
                    In the initial transition phase from a symbiotic bacterium to an organelle, the proto-mitochondrial genome must have encoded more than a thousand genes specifying mitochondrial proteins, and must have adhered to bacterial conventions with respect to genome organization, RNA transcription and protein translation. However, in present-day mitochondria from animals, fungi and plants, gene organization and gene expression bear little traces of their bacterial past. 
                    <br>
                    This project aims at elucidating the steps that had been involved in the 'domestication' of the endosymbiotic bacterial ancestor, and at tracing the processes by which the predecessor of mitochondria was transformed into a eukaryotic organelle. These questions can now be tackled effectively through our discovery of mitochondrial DNAs resembling bacterial genomes in miniature: in unicellular, flagellated protozoans called jakobids.
                    <br>
                    We are investigating <i>Andalucia godoyi</i> to find out the degree to which mitochondrial information processing --from gene via RNA to protein-- resembles that of bacteria or rather 'modern' mitochondria as we known them from animals, fungi, and plants. Our approach combines classical biochemistry, new generation genomics methodologies and bioinformatics. For example, we investigated by proteomics the structure of the <i>Andalucia</i> mitoribosome, revealing an unexpected complex machinery in this protist with the most bacteria-like mito-ribosomal RNAs (Valach et al 2021 Molec Biol Evol) (NSERC RGPIN-2014-05286)
                </p>

                <div class="theme">
                    <img src="image/th6.png" alt="image7">
                    <p class="project_name">
                        Ongoing Bioinformatic tool development
                    </p>
                </div>
                <p class="project_text">
                    I am participating in the development of automated genome annotation tools such as <a href="https://megasun.bch.umontreal.ca/RNAweasel/">MFannot</a>, an organelle genome annotator, and <b>Eukan</b>, a nuclear genome annotator that will soon be released. These projects are spearheaded by BF Lang. My contribution consists in testing the tools for robustness (particularly when using unusual genome architectures and gene structures), performance, ease of usage and informative output. (NSERC, FRQNT 2018-PR-206806)
                </p>

            </div>

            <h2 class="secondary_pages"> Past projects</h2>
            <div class="past_projects">

                <p class="project_name">
                    Genomics of bacterial endosymbionts in eukaryotes
                </p>
                <p class="project_text">
                    <b>CINAR Pathobacter</b> was a European Union Research Project. In collaboration with four other universities in Europe and one in the USA, we are investigating ciliate-bacteria relationships at an ecological, functional and evolutionary-genomics level. The Montreal group (BF Lang & G Burger) was focussing on genome sequencing and annotation of <i>Holospora</i> bacteria.

                </p>

                <p class="project_name">
                    Omics for Phytoremediation 
                </p>
                <p class="project_text">
                    The <b>GenoRem</b> project aimed at studing the inter-relationships between plants, fungi, and bacteria in the detoxification of soils that are contaminated with organic or inorganic compounds. The experimental approaches include genomics, transcriptomics, proteomics, and metabolomics. Bioinformatics methodologies assure thorough data management and analysis. This project is conducted by ~30 collaborating researchers of the Universite de Montreal and McGill University, and is financed by Genome Canada and Genome Quebec. Long-term goal is to select optimal combinations of plants and microorganisms for particular soil contaminations.
                </p>

                <p class="project_name">
                    Nuclear Genome Exploration 
                </p>
                <p class="project_text">
                    The genome project <b>Unicorn</b> was a collaboration of seven research groups from Canada, UK, and USA, 
                    and endorsed by the National Human
                    Genome Research Institute (NHGRI). This project aimed at understanding
                    how multicellularity first evolved. Genomic data were generated from
                    unicellular relatives of animals and fungi, i.e., choanflagellates,
                    Ichthyosporea, Nuclearidae, chytrids, zygomycetes and apusozoa (outgroup) (see e.g., Ruiz-Trillo I et al, 2007 Trends Genet).
                </p>

                <p class="project_name">
                    Organelle Genome Database
                </p>
                <p class="project_text">
                    <b>GOBASE</b> was a comparative database that tied together and unified the 
                    various data on mitochondrial and chloroplast genomes and the organism which contain 
                    them, by making the information network-accessible to the scientific community.
                    Data validation and addition of missing information is at the center
                    of this project (O'Brien E et al 2007 Nucleic Acids Res).
                </p>

                <p class="project_name">
                    EST Surveys 
                </p>
                <p class="project_text">
                    The Protist EST Program 
                    <b><a href="http://megasun.bch.umontreal.ca/pepdb/pep.html">(PEP)</a></b>, 
                    a collaboration involving eight Canadian 
                    research groups, aims to determine
                    the expressed portions of genomes from a taxonomically broad collection
                    of mostly unicellular eukaryotes. The organismal group studied
                    by myself and my collegues 
                    BF. Lang and MW. Gray are jakobid flagellates, which are believed to be amongst
                    the most primitive extant eukaryotes.
                    The goal of this project is to better understand early eukaryotic cell evolution
                    (Keeling PJ et al, 2005 Trends Ecol Evol). See also TBestDB below.
                </p>

                <p class="project_name">
                    Previous Bioinformatics Developments 
                </p>
                <p class="project_text">
                    <B>TBestDB</B>
                    was a database to organize c-DNA and EST data from poorly-investigated protistan eukaryotes, 
                    generated by the pan-Canadian Protist EST program. Cross-referencing will 
                    be possible between the various PEP projects and with data from model 
                    organisms, including yeast, flatworm, but also cyanobacteria and 
                    alpha-proteobacteria, the ancestors of eukaryotic organelles. Interoparability 
                    with single-organism databases will be possible via the 
                    gene ontology framework (developped by the GO consortium).
                    Released data are deposited into TBestDB for network access and download by 
                    the scientific community (O'Brien E et al, 2007). 

                    <br>
                    <b>AutoFACT</b> is 
                    an automated pipeline to annotate EST sequences in a comprehensive and
                    informative way. It hsd been used to annotate PEP data, and 
                    available as open source.
                    <br>
                    <b>AnaBench</b>, 
                    a web-based, integrated analysis environment, provides biologists access 
                    to diverse bioinformatics tools. The current prototype, which is accessible to 
                    the public, includes translation, Blast searches, multiple alignment, 
                    tRNA search, and more.

                    <br>
                    <b>Prediction of protein function and cellular localization</b>. In typical 
                    genome projects, only ~50% of the protein-coding genes can be assigned to a function, 
                    and even less to a particular cellular location. This highlights the need of 
                    sensitive and efficient prediction methods. 
                    The main objective of this project is to apply machine learning methods 
                    (predictive data mining), to detect hidden signatures and patterns in integrated 
                    biological data, and to employ this new knowledge for deciphering genomic data at 
                    a large scale (Shen & Burger, 2007 BCM Bioinformatics; Kannan & Burger, 2008 BCM Genomics, ProteinPep Lett). 

                </p>

                <p class="project_name">
                    Organelle Genome Megasequencing Program 
                </p>
                <p class="project_text">
                    <a HREF="http://megasun.bch.umontreal.ca/ogmp/welcome.html">(OGMP)</a><br>
                    The OGMP is a  collaborative project aiming at complete sequencing of mitochondrial and chloroplast 
                    genomes of a phylogenetically broad collection of mostly unicellular 
                    eukaryotes (Protista) (see Gray et al 2004, Annu Rev Genet).
                </p>
            </div>

        </div>


    </div>
    
    <footer>
        <div class="overfooter">
            <div class="overlay2"></div>
            <p class="contact">
                CONTACT <br>
                Université de Montréal <br>
                Robert-Cedergren Center in Bioinformatics & Genomics<br>
                Département de Biochimie<br>
                Pavillon J. Armand Bombardier<br>
                2900 Blvd Edouard-Montpetit<br>
                Montréal, Québec, Canada, H3T 1J4<br>
                email: gertraud.burger[AT]umontreal.ca &nbsp&nbsp&nbsp&nbsp where [AT]=@
            </p>
            <p class="design">© 2025 Gertraud Burger. Made by mariamadb[at]hotmail.com</p>
        </div>
        
    </footer>

    <script>
        document.addEventListener("DOMContentLoaded", () => {
          const faders = document.querySelectorAll(".fade-in");
      
          const observer = new IntersectionObserver((entries, obs) => {
            entries.forEach(entry => {
              if (entry.isIntersecting) {
                entry.target.classList.add("visible");
                obs.unobserve(entry.target);
              }
            });
          }, {
            threshold: 0.1
          });
      
          faders.forEach(el => observer.observe(el));
        });


        function toggleMenu() {
            const navbar = document.querySelector('.navbar2');
            const toggleBtn = document.getElementById('menuToggleBtn');

            navbar.classList.toggle('active');

            if (navbar.classList.contains('active')) {
                toggleBtn.textContent = '✖'; 
            } else {
                toggleBtn.textContent = '☰'; 
            }
            }
        function closeMenuOnResize() {
        const navbar = document.querySelector('.navbar2');
        const toggleBtn = document.getElementById('menuToggleBtn');
        if (window.innerWidth > 768) {
            navbar.classList.remove('active');
            toggleBtn.textContent = '☰'; 
        }
        }
        window.addEventListener('resize', closeMenuOnResize);
      </script>
      


</body>
</html>
